### ISSUE TITLE
Concurrency issue when setting custom variables

### ISSUE THREAD

**Issue by d4rken** [piwik/piwik-sdk-android#89]
Seems to be very rare, happened only a few times out of a few 10k events.
Writing this down so I don't forget it :).
Maybe fix this and do #69 at the same time?

```
java.util.ConcurrentModificationException
        at java.util.HashMap$HashIterator.nextEntry(HashMap.java:806)
        at java.util.HashMap$EntryIterator.next(HashMap.java:843)
        at java.util.HashMap$EntryIterator.next(HashMap.java:841)
        at org.json.JSONObject.<init>(JSONObject.java:128)
        at org.piwik.sdk.CustomVariables.toString(CustomVariables.java:90)
        at org.a.a.e.a(Tracker.java:16684)
        at eu.thedarken.sdm.tools.e.a.a(PiwikHelper.java:13159)
        at eu.thedarken.sdm.tools.piwik.PiwikHelper$1.run(PiwikHelper.java:305)
        at java.lang.Thread.run(Thread.java:841)
```

**Comment by dotsbb** [comment]
I would like to take care of it, if You don't mind

**Comment by d4rken** [comment]
Sure :+1:, happy hacking.

**Comment by d4rken** [comment]
Fixed by @dotsbb in #90
